### PR TITLE
Messages tests

### PR DESCRIPTION
**Pull request by shakedhagag**
#PR-Description

Finished tests for messages classes.

### CODE REVIEW

**Review by DanWaldman — APPROVED** [review]
great

**Comment by DanWaldman** [comment]
Resolved #41 #44 #46

**Review by razashtar93 — APPROVED** [review]
LGTM

**Review by DanielItkes — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (Yarboa) [inline .vscode/settings.json:1]: Please use git rm --cache .vscode
Before the git commit

File: AcadeMeData/tests.py [hunk 2]
```diff
@@ -47,51 +35,116 @@ def test_get_user(self):
...
+        return message
+
+    def test_get_msg(self, msgID=1, userID=None, text='This is a test message yo'):
+        userID = self.user_example()
+        msg = TestMessagesModel.generate_message(self, msgID, userID, text)
+        msg_test = Messages.get_msg_by_id(1)
+        assert msg_test == msg
+        assert isinstance(msg_test, Messages)
+
+    def test_create_msg(self, msgID=9, text='I am testinggggg'):
```
Comment (Yarboa) [inline AcadeMeData/tests.py:128]: I there a way to delete messages?

File: .vscode/settings.json (hunk 1 above)
Comment (shakedhagag) [reply .vscode/settings.json:1]: Why is clearing the cache needed?

File: AcadeMeData/tests.py (hunk 2 above)
Comment (shakedhagag) [reply AcadeMeData/tests.py:128]: Not right now, it slipped off my mind, will add it later.

**Pull request closed without merging**